src/runner/func_runner.cpp

```diff
@@ -121,10 +121,14 @@ int main(int argc, char* argv[])
     storage::initFaasmS3();
     faabric::transport::initGlobalMessageContext();
 
+    PROF_BEGIN
```

Comment:
I see the argument against committing code to mark up specific operations, as these may not be useful to everyone doing profiling. I personally think it's useful to leave the profiling markup around operations that _risk_ being slow in future. If someone is profiling the application it's probably because it's slow, and having existing profiling annotations around risky code is like giving that person a nudge in future and saying "make sure you confirm that xyz is still performant".

This markup is different as it's the main wrapper code that will be useful to everyone profiling this executable. Without it, the profiling information won't be printed and I found it confusing to remember what I needed to add and where.